src/app/components/dynamic-list/DynamicListItem.jsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import PropTypes from "prop-types";
+import PersonInShield from "../../icons/PersonInShield";
+import TickInShield from "../../icons/TickInShield";
+import Person from "../../icons/Person";
+
+const propTypes = {
+    title: PropTypes.string,
+    desc: PropTypes.string,
+    icon: PropTypes.oneOf(["Cross", "Person", "PersonInShield", "TickInShield", "Warning"]),
+    iconColor: PropTypes.oneOf(["standard", "blue", "green", "red"]),
+    iconName: PropTypes.string,
+    buttonName: PropTypes.string,
+    buttonCallback: PropTypes.func,
+};
+
+const DynamicListItem = props => {
+    const colorBlack = "#000000";
+    let icon;
+    switch (props.icon) {
+        case "Shield-person":
+            icon = <PersonInShield classes="dynamic-list-item__icon" viewBox="0 0 15 15" fillColor={colorBlack} ariaLabel={props.iconName} />;
+            break;
+        case "Shield-tick":
+            icon = <TickInShield classes="dynamic-list-item__icon" viewBox="0 0 15 15" fillColor={colorBlack} ariaLabel={props.iconName} />;
+            break;
+        case "Person":
+            icon = <Person classes="dynamic-list-item__icon" viewBox="0 0 15 15" fillColor={colorBlack} ariaLabel={props.iconName} />;
+            break;
+    }
+    return (
+        <li className="dynamic-list-item">
+            {icon}
+            <span className="dynamic-list-item__title">{props.title}</span>
```

Comment:
Similar to in the `DynamicList` component, should we be using a heading element here instead of a `span`?

Reply:
I think in this instance the title isn't a heading per se, might be that 'title' isn't an accurate word to use here (not sure what would be a better fit for 'main list item text field'. 

I think this item is styled similar to a heading; bigger and bolder. But it doesn't semantically section content off in the hierarchy of page content. It can be that there is just a list of these headings but no content that follows each of them.

In addition, this component could be used as a subsection of a multitude of heading levels (depending on where on the page it is used, could be nested under a h3 meaning it should be an h4 (to keep the order of headings correct). So each time it is used we would have to pass in a prop and the parent would have to declare what level it should be.

I also think by doing this screen readers may get confused thinking there are 2000+ section on this page where really this is just a list of results (we currently have 1800 Florence users which will be in this list).

Reply:
Yeah, could be quite messy with that many users. Think it's fine to leave as is for now then as it's not so much a title as you say